manifests/kube/kube.json

```diff
@@ -1,40 +1,46 @@
 {
   "name": "kube",
-  "description": "A plugin to manage spin apps",
+  "description": "Commands for publishing applications to Kubernetes",
   "homepage": "https://github.com/spinkube/spin-plugin-kube",
-  "version": "0.0.1",
+  "version": "0.1.0",
   "spinCompatibility": ">=1.5",
```

Comment:
@karthik2804 wanted to know if we’ve actually tested this plugin against Spin 1.5.

@rajatjindal have you tested this plugin against spin 1.5? If not we should update this to a more recent version.

Reply:
I've not tested this with spin 1.5. we should change it to 2.3.1 as we have a OCI push fix in that version?

Reply:
I've pushed a commit to have min spin version 2.3.1